eds4jinja2/adapters/base_data_source.py

```diff
@@ -45,8 +49,11 @@ def fetch_tabular(self) -> Tuple[Optional[object], Optional[str]]:
             if not self._can_be_tabular():
                 raise UnsupportedRepresentation("Only TREE representation is supported")
             else:
-                return self._fetch_tabular(), None
+                result = self._fetch_tabular()
+                logger.info(f"With {type(self).__name__}, fetching as tabular - {str(self)}", extra={'class': self.__class__.__name__})
```

Comment:
it seems that 
```python
type(self).__name__
# and
self.__class__.__name__
```
achieve the same thing, why not use only one version?

Reply:
deal, that was a mistake